.gitignore

```diff
@@ -26,3 +26,4 @@ pnpm-lock.yaml
 *.njsproj
 *.sln
 *.sw?
+tsconfig.tsbuildinfo
```

Comment:
MINOR:

It would be nice to add a new line to the end of the file

Reply:
Pushed with a new line, mind if I ask why though? 😛

Reply:
Did you manage to test version 5 with any sketches of your own or otherwise, would appreciate the feedback if you did, I tried with some of mine but simple use cases only so far.

Reply:
As for the new line, it's specific to git, how the diff utility is implemented. That's why github highlighted it. Read more: https://unix.stackexchange.com/a/18744

We can add an `editorconfig` file to the repository to avoid this in the future: https://editorconfig.org/#example-file

Reply:
I haven't tested it locally, but I may do so today / tomorrow

Reply:
Okay, I'll wait for you to test and give feedback then before merging.

Reply:
I think editorconfig is good but prettier would be better, maybe we can automate that somehow, a custom rule or simple plugin for example.